kpops/cli/main.py

```diff
@@ -130,37 +144,51 @@ def get_step_names(steps_to_apply: list[PipelineComponent]) -> list[str]:
     return [step.name.removeprefix(step.prefix) for step in steps_to_apply]
 
 
-def filter_steps_to_apply(
-    pipeline: Pipeline, steps: set[str]
-) -> list[PipelineComponent]:
-    skipped_steps: list[str] = []
+def filter_component(
+    pipeline: Pipeline, steps: set[str], filter_type: FilterType
+) -> list:
```

Comment:
please don't omit the value type in the return type annotation

Reply:
done